sdks/python/apache_beam/yaml/tests/assign_timestamps.yaml

```diff
@@ -82,3 +82,118 @@ pipelines:
           config:
             elements:
               - {user: bob, timestamp: 3}
+
+# Assign timestamp to beam row element with error handling and output schema
+# check.
+  - pipeline:
+      type: composite
+      transforms:
+        - type: Create
+          name: CreateVisits
+          config:
+            elements:
+              - {user: alice, timestamp: "not-valid"}
+              - {user: bob, timestamp: 3}
+        - type: AssignTimestamps
+          input: CreateVisits
+          config:
+            timestamp: timestamp
+            error_handling:
+              output: invalid_rows
+            output_schema:
+              type: object
+              properties:
+                user:
+                  type: string
+                timestamp:
+                  type: integer
+              error_handling:
+                output: invalid_schema_rows
```

Comment:
Haven't gotten to the point where we actually implement this, but it seems to me like it might be more useful to just have a single error_handling output associated with a transform where we capture all problematic records. In most cases, a user is going to want to write these somewhere for manual inspection/reprocessing and I don't think it matters too much whether it is because the output is an error or the output doesn't match the expected schema.

Reply:
This could be done with a flatten step where you unify the error output from schema validation and normal exception handling.

Reply:
In practice, they're both probably going to be data issues which users want to pipe to some exception handling sink, so I think treating them as the same collection probably makes sense.

Reply:
Will discuss with group also to finalize.